lib/pr_checker.js

```diff
@@ -168,6 +170,32 @@ class PRChecker {
       }
     }
 
+    if (isFastTracked) {
+      const comment = this.comments.find((c) =>
+        FAST_TRACK_RE.test(c.bodyText));
+      if (!comment) {
+        cli.error('Unable to find the fast-track request comment.');
+        return false;
+      }
```

Comment:
This is probably OK but just in case: There is a window, often of several minutes, where an issue is labeled with `fast-track` but the comment has not yet appeared. So this error may happen legitimately. Maybe that's fine? Or maybe we just need to mention it in the error message (perhaps by adding "Please try again in a few minutes." or something like that?).

Reply:
If there is no comment it means that there is no approval and therefore the PR can not be landed. So I think it's useless.